ydb/core/viewer/viewer_ut.cpp

```diff
@@ -1831,4 +1832,107 @@ Y_UNIT_TEST_SUITE(Viewer) {
         auto resultSets = json["Databases"].GetArray();
         UNIT_ASSERT_EQUAL_C(1, resultSets.size(), response);
     }
+
+    static const ui32 ROWS_N = 15;
+    static const ui32 ROWS_LIMIT = 5;
+
+    TString PostExecuteScript(TKeepAliveHttpClient& httpClient, TString query) {
+        TStringStream requestBody;
+        requestBody
+            << "{ \"database\": \"/Root\","
+            << " \"script_content\": {"
+                << " \"text\": \"" << query << "\"},"
+            << " \"exec_mode\": \"EXEC_MODE_EXECUTE\" }";
+        TStringStream responseStream;
+        TKeepAliveHttpClient::THeaders headers;
+        headers["Content-Type"] = "application/json";
+        headers["Authorization"] = "test_ydb_token";
+        const TKeepAliveHttpClient::THttpCode statusCode = httpClient.DoPost(TStringBuilder()
+                                                            << "/query/script/execute?timeout=600000"
+                                                            << "&database=%2FRoot", requestBody.Str(), &responseStream, headers);
```

Comment:
I know that dublicate database parameters in body and uri doesn't look nice. I would change it in another commit.